tests/test-node.py

```diff
@@ -54,7 +54,10 @@ def test_node_scontrol():
 
 def test_node_update():
     """Node: Test node().update()."""
-    node_test_before = pyslurm.node().find_id("c10")
+    import time
```

Comment:
`import` statements should be in test methods.

Reply:
Sorry, but I don't get the point?

Reply:
Sorry, meant should *not*

Reply:
Yes, you are right.